test/protobuf/conformance_regressions_test.exs

```diff
@@ -94,6 +94,10 @@ defmodule Protobuf.ConformanceRegressionsTest do
   end
 
   # Fails on OTP 25.
+  if :erlang.system_info(:otp_release) >= ~c"25" do
```

Comment:
Does it fail in all Elixir versions with OTP 25? What do we know so far?

Reply:
Yes, it seems to be a compatibility issue with OTP 25, rather than an issue we are having with Elixir.